chronograf.go

```diff
@@ -32,12 +33,13 @@ func (e Error) Error() string {
 type Logger interface {
 	Debug(...interface{})
 	Info(...interface{})
-	Warn(...interface{})
```

Comment:
Okay, I didn't have a clear idea for what they'd be used for, but I like your guidelines. Agree with ✂️ until we have a use for it.